src/basic_recipes/axis.jl

```diff
@@ -317,8 +317,8 @@ function draw_axis3d(textbuffer, linebuffer, scale, limits, ranges_labels, fonts
     return
 end
 
-function plot!(scene::SceneLike, ::Type{<: Axis3D}, attributes::Attributes, args...)
-    axis = Axis3D(scene, attributes, args)
+function plot!(axis::Axis3D)
+    scene = parent_scene(axis)
```

Comment:
`parent_scene` gives me the expectation that the axis is contained in some scene that is returned/used here. Maybe it would be better to use `get_scene` to avoid that?